apps/src/netsim/NetSimRouterLogModal.html.ejs

```diff
@@ -30,6 +45,69 @@ var showPacketInfo = headerFields.indexOf(Packet.HeaderType.PACKET_INDEX) > -1 &
  * @type {boolean}
  */
 
+function routerLogModeToggleButton() {
+  if (!locals.canToggleRouterLogMode) {
+    return '';
+  }
+
+  var markup = '<button type="button" id="routerlog-toggle" class="pull-right btn btn-primary btn-mini">';
+  if (locals.isAllRouterLogMode) {
+    markup += i18n.logBrowserHeader_toggleMine()
+  } else {
+    markup += i18n.logBrowserHeader_toggleAll()
+  }
+  markup += '</button>';
```

Comment:
Do we get es6 in ejs files? If so, this would be a great spot for string interpolation!

```javascript
let headerText = locals.isAllRouterLogMode
  ? i18n.logBrowserHeader_toggleMine()
  : i18n.logBrowserHeader.toggleAll();

let markup = `<button type="button" id="routerlog-toggle" class="pull-right btn btn-primary btn-mini">${ headerText }</button>`
```